src/app.js

```diff
@@ -142,9 +142,10 @@ router.post('/action', verifyRequest, actionHandler(async (action) => {
 		case 'block_actions': {
 			const [answer, date] = action.actions[0].value.split('_');
 			const channel = action.channel.id;
+			const slackId = action.container.message_ts;
 			const timestamp = Date.now();
 			logger.debug({user, channel, date, answer, timestamp}, 'Vote action');
-			await dynamo.vote({channel, date, user, answer, timestamp});
+			await dynamo.vote({channel, date, user, answer, timestamp, slackId});
```

Comment:
Is this necessary? I doesn't look like you've modified `dynamo.js` in this PR so `slackId` will just be ignored.

Reply:
Yup, that is from a solution long gone. Bad clean up and spot checking on my part.